contracts/satelites/PullPayment.sol

```diff
@@ -83,7 +83,8 @@ contract PullPayment is Ownable {
       lastDay = now;
       spentToday = 0;
     }
-    if (spentToday.add(amount) > dailyLimit || spentToday.add(amount) < spentToday) {
+    // not using safe math because we don't want to throw;
+    if (spentToday + amount > dailyLimit || spentToday + amount < spentToday) {
```

Comment:
what if spentToday + amount will overflow? I think the method will return `true` which is not correct